packages/shared/world/SceneWorker.ts

```diff
@@ -21,17 +15,88 @@ export enum SceneWorkerReadyState {
   DISPOSED = 1 << 8
 }
 
+import { registerServices } from 'shared/apis/host'
+import { PortContext } from 'shared/apis/host/context'
+import { pushableChannel } from '@dcl/rpc/dist/push-channel'
+import Protocol from 'devtools-protocol'
+
+function createGenericLogComponent() {
```

Comment:
Minor: Some reason this is here in `SceneWorker`? Should not be `createSceneWorkerLogComponent` if only is for the `SceneWorker`?